portfolio/src/main/webapp/reviews.html

```diff
@@ -29,23 +30,44 @@
 
     <div class="parallax-background-image black-background-image">
       <div class="titlecontainer">
-        <span class="homepageheader1">~Food Reviews~</span>
-        <span class="homepageheader2">Gimme your love, or your roasts xD</span>
+        <span class="homepage-header-1">~Food Reviews~</span>
+        <span class="homepage-header-2">Gimme your love, or your roasts xD</span>
       </div>
     </div>
 
     <section class="section section-dark">
-      <button onclick="getReviews()">Gimme some reviews!</button>
-      <div id="reviews-container"></div> 
+      <div class="user-review-container">
+        <form action="/reviews" method="POST">
+          <div class="row">
+            <div class="col-25">
+              <label for="reviewer-name">Name</label>
+            </div>
+            <div class="col-75">
+              <input type="text" id="reviewer-name" name="reviewer-name" placeholder="Your name..">
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-25">
+              <label for="reviewer-input">Review</label>
+            </div>
+            <div class="col-75">
+              <textarea id="reviewer-input" name="reviewer-input" placeholder="Review here.." style="height:200px"></textarea>
+            </div>
+          </div>
+          <div class="row">
+            <input type="submit" value="Submit" class="button submit-button">
+          </div>
+        </form>
+      </div>
+
+      <label for="max-comments"># of Comments (between 1 and 5):</label>
+      <input type="number" id="max-comments" name="max-comments" min="1" max="5"><br>
+      <button onclick="getReviews()" class="button recent-button">Most Recent Reviews!</button><br>
+      <form action="/delete-data" method="POST">
```

Comment:
hey qq, is this supposed to be in this PR? I'm wondering if it's an artifact from the split PRs because this seems to be about fetching the n most recent.